versions-shipped.json

```diff
@@ -1 +1,156 @@
-[]
+[
+  "1.0.0",
```

Comment:
@Eomm these version were generated when i ran `npm run build:website` 

wondering if i missed something to ignore them....or they are supposed to be removed manually

Reply:
> wondering if i missed something to ignore them....or they are supposed to be removed manually

Right now they should be removed manually from the commit - will find a better approach with #46